test/spec/modules/precisoBidAdapter_spec.js

```diff
@@ -145,7 +145,7 @@ describe('PrecisoAdapter', function () {
     })
   })
   describe('getUserSyncs', function () {
-    const syncUrl = 'https://ck.2trk.info/rtb/user/usersync.aspx?id=NA&gdpr=0&gdpr_consent=&us_privacy=&t=4';
+    const syncUrl = 'https://ck.2trk.info/rtb/user/usersync.aspx?id=12355454test&gdpr=0&gdpr_consent=&us_privacy=&t=4';
```

Comment:
I am confused how this test was needing to be updated since the change is only on the `buildRequests` part, not the `getUserSyncs part.

Reply:
Sorry for the confusion. We added the same userId into the usersync URL, so we also included it in the test case. We apologize for not mentioning that change in this PR.